tests/baselines/reference/api/typescript.d.ts

```diff
@@ -2821,8 +2821,8 @@ declare namespace ts {
             private hasAddedOrRemovedSymlinks;
             protected languageService: LanguageService;
             languageServiceEnabled: boolean;
-            readonly trace?: (s: string) => void;
-            readonly realpath?: (path: string) => string;
+            trace?(s: string): void;
+            realpath?(path: string): string;
```

Comment:
So why did these need to switch away from `readonly` (and to methods, presumably to exempt some code from `readonly` checks)?